### PR TITLE
[11.0][MIG] product_state

### PR DESCRIPTION
**Pull request by asaunier**
Simple port of https://github.com/OCA/product-attribute/tree/10.0/product_state to Odoo 11.

### CODE REVIEW

File: product_state/views/product_views.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <field name="arch" type="xml">
     	    <field name="type" position="after">
                 <field name="state"/>
-	    </field>
+            </field>
```
Comment (pedrobaeza) [inline product_state/views/product_views.xml]: Why this extra space?

Comment (asaunier) [reply product_state/views/product_views.xml]: Good catch, thanks :)
Actually it is not an extra space: I had converted a tab to 4 spaces and it seems github does not show them the same way. Your comment made me realize the opening ``<field>`` tag (line 22) was not correctly indented either => I have just pushed 0d332ec making sure the whole file has no tab.

**Review by pedrobaeza — APPROVED** [review]

**Review by p-tombez — APPROVED** [review]
LGTM, didn't review i18n

**Review by tschanzt — APPROVED** [review]

File: product_state/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
```
Comment (grindtildeath) [inline product_state/README.rst]: 11.0

File: product_state/README.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Product State
+=============
+
+In version 10.0 Odoo removes the state field from product. This module re-introduces it and allows simple product life cycle.
```
Comment (grindtildeath) [inline product_state/README.rst]: IMO you should get rid of the first part and have something like : 'This module introduces the state field on product template and allows simple product life cycle.'

File: product_state/README.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Cedric Pigeon <cedric.pigeon@acsone.eu>
```
Comment (grindtildeath) [inline product_state/README.rst:41]: Add yourself ?

File: product_state/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Product State",
+    'description': """
+        Module introducing a state field on product template""",
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (grindtildeath) [inline product_state/__manifest__.py]: https://github.com/OCA/product-attribute

File: product_state/README.rst [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Product State
+=============
+
+In version 10.0 Odoo removes the state field from product. This module re-introduces it and allows simple product life cycle.
+
+Installation
```
Comment (grindtildeath) [inline product_state/README.rst]: You can remove Installation and Configuration sections.

File: product_state/README.rst [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/product-attribute/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/product-attribute/issues/new?body=module:%20product_life_period%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (grindtildeath) [inline product_state/README.rst]: Please update this section considering : https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Review by grindtildeath — COMMENTED** [review]
Out of README and __manifest__ to update :+1:

**Comment by asaunier** [comment]
@grindtildeath Thanks! Updated.

**Review by guewen — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash your last 2 commits before merging.

**Comment by asaunier** [comment]
@pedrobaeza Squashed!
Thanks all for reviewing!

**Pull request merged**